eo-maven-plugin/src/main/java/org/eolang/maven/Home.java

```
@@ -0,0 +1,189 @@
+            return bytes;
+        } catch (final IOException ex) {
+            throw new IOException(
+                String.format(
+                    "Failed while trying to save to %s",
+                    this.rel(path)
+                ),
+                ex
+            );
+        }
+    }
+
+    /**
+     * Saving string.
+     * @param str String
+     * @param path Path
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final String str, final Path path) throws IOException {
+        return this.save(new InputOf(str), path);
+    }
+
+    /**
+     * Saving text.
+     * @param txt Text
+     * @param path Path
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final Text txt, final Path path) throws IOException {
+        return this.save(new InputOf(txt), path);
+    }
+
+    /**
+     * Saving stream.
+     * @param stream Input stream
+     * @param path Path
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final InputStream stream, final Path path) throws IOException {
+        return this.save(new InputOf(stream), path);
+    }
+
+    /**
+     * Saving bytes.
+     * @param bytes Byte array
+     * @param path Path
+     * @return Saved bytes
+     * @throws IOException If fails
+     */
+    public long save(final byte[] bytes, final Path path) throws IOException {
+        return this.save(new InputOf(bytes), path);
+    }
+
+    /**
+     * Saving string.
+     * @param str String
+     * @param path Path
+     * @return Saved bytes
+     * @todo #1105:30min We should remove this function and get rid of using it
+     *  because it is not really better than simple save
+     */
+    public long saveQuietly(final String str, final Path path) {
+        try {
+            return this.save(new InputOf(str), path);
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
```

I don't think it is a good practice to catch one type of exception and throw another one. The stack trace of the original exception is lost.

---

This code is not mine, it was pasted from previous implementation, but I got your point. We want to remove this method at all in future. You may also see `todo`